kofta/src/vscode-webview/svgs/GitHubIcon.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from "react";
+
+/* SVG from iconmonstr, https://iconmonstr.com/license/ */
```

Comment:
you could get them from their official "branding" page though

Reply:
Or from the `react-feather` library, which is already in use.